run_brer/run_config.py

```diff
@@ -369,12 +369,15 @@ def __train(self, tpr_file=None, **kwargs):
             current_alpha = context.potentials[i].alpha
             # noinspection PyUnresolvedReferences
             current_target = context.potentials[i].target
+            current_stop_called = context.potentials[i].stop_called
 
             self.run_data.set(name=current_name, alpha=current_alpha)
             self.run_data.set(name=current_name, target=current_target)
-            self._logger.info("Plugin {}: alpha = {}, target = {}".format(current_name,
-                                                                          current_alpha,
-                                                                          current_target))
+            self.run_data.set(name=current_stop_called, stop_called=current_stop_called)
```

Comment:
I'm not sure you can do this without updating the lists of named data. I think that's in the `run_data.py` module.

Reply:
Pausing on this issue to examine `run_data.py`

Reply:
I'm looking into it.